### PR TITLE
Installation doc updated for version 2.1 (#372)

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #372 for version 2.1.

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -1,29 +1,128 @@
 # Installation
 
-The latest stable version of _taipy-gui_ is available through _pip_:
+There are different ways to install Taipy GUI, depending on how
+you plan to use it.
+
+If your goal is to look into the code, modify and improve it, go straight
+to the [source installation](#installing-for-development) section.
+
+Taipy GUI needs your system to have Python 3.8 or above installed.
```
Comment (FredLL-Avaiga) [inline INSTALLATION.md:9]: Taioy GUI relies on Python 3.8 or above ?

File: INSTALLATION.md [hunk 2]
```diff
@@ -1,29 +1,128 @@
...
+Run the command:
+```console
+$ pip install taipy-gui
 ```
-pip install taipy-gui
+
+If you are running in a virtual environment, you will have to
+issue the command:
+```console
+$ pipenv install taipy-gui
```
Comment (FredLL-Avaiga) [inline INSTALLATION.md:24]: mention conda ?

**Review by FredLL-Avaiga — APPROVED** [review]
Have I already reviewed this ?

File: INSTALLATION.md (hunk 1 above)
Comment (FabienLelaquais) [reply INSTALLATION.md:9]: Well... that's what the setup.py says: `python_requires=">=3.8"`
As well as the doc and the Web site...

File: INSTALLATION.md (hunk 2 above)
Comment (FabienLelaquais) [reply INSTALLATION.md:24]: We're not ready yet for explaining a proper Conda installation.
That needs to be done soon enough though. Good point.

**Pull request merged**